### DISCUSSION TITLE
Full Stack FastAPI /w MongoDB ?

### DISCUSSION THREAD

**Discussion by echatzikyriakidis** [fastapi/fastapi#7204] [Questions] [answered]
Hi to all!

I would like to ask if anyone knows how this is going:

https://fastapi.tiangolo.com/project-generation/#full-stack-fastapi-mongodb

I would like to use in the near future a solution like that.

Thank you and stay safe.

**Comment by Kludex** [comment]
Well, I doubt that is going to happen any time soon. That section didn't change in more than 6 months...

What you can do for now is check if there are any unofficial projects. Actually, you can Google my "awesome FastAPI projects" (which I don't recommend, is not up-to-date and is ugly) and you can see a list of projects using MongoDB, maybe some are production ready.

--

If by any chance someone wants to help on that, my suggestion is:
- Create a project analogous to the other "production projects"
- Create an issue and post the progress (community input can be useful)

Once finished, wait for @tiangolo, and at some point he will check it. Possible scenarios:
- not good enough, not going to be recommended in the docs
- good enough, it will be forked, some changes applied and docs added

**Note:** those are my thoughts, don't assume everything that I said as absolute truth!

**Comment by echatzikyriakidis** [comment]
Thanks!

**Comment by tiangolo** [comment, marked as answer]
Thanks for the help @Kludex! ☕ 

@echatzikyriakidis I would recommend ODMantic: https://art049.github.io/odmantic/

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
